### DISCUSSION TITLE
Dockerfile: create directoriy / chowner fails on Debian server  but works fine in Docker Desktop

### DISCUSSION THREAD

**Discussion by mhackerts** [docker-solr/docker-solr#411] [Q&A]
Hi, here's my Dockerfile:

_FROM solr:8
USER 0
RUN apt-get update
RUN mkdir -p /var/solr/data/democore/data
RUN chown -R solr:solr /var/solr/data/democore/data
USER 8983_

I created and tested Dockerfile on my local Docker Desktop, works fine.
When I run on real Debian server I get error on line running chown, stating that directory /var/solr/data/democore/data does not exist.
Any Idea? (sorry for that this may not be a solr-related problem but rather docker / linux)

Heres what I get from build:
_Step 5/9 : RUN mkdir -p -v /var/solr/data/democore/data
 ---> Running in f7404885d47d
mkdir: created directory '/var/solr/data/democore'
mkdir: created directory '/var/solr/data/democore/data'
Removing intermediate container f7404885d47d
 ---> 0e456ef71a8f
Step 6/9 : RUN chown -R solr:solr /var/solr/data/democore/data
 ---> Running in 5a63119ecf43
chown: cannot access '/var/solr/data/democore/data': No such file or directory
ERROR: Service 'solr' failed to build: The command '/bin/sh -c chown -R solr:solr /var/solr/data/democore/data' returned a non-zero code: 1_

**Comment by dsmiley** [comment]
I don't have a complete answer for you but I suspect the problem is related to /var/solr/data being a VOLUME.
